### DISCUSSION TITLE
Provide a jar of mcs

### DISCUSSION THREAD

**Discussion by maxandersen** [mthmulders/mcs#305] [Ideas]
Today you must download specific binaries for your specific OS build. But some OS don't have matching arch's and it's different mechanics on each OS one need to use. 

Would be nice if in addition to the OS specific binaries also provided a .jar artifact. Either in GitHub releases or maven central (or both). 

Then no matter what OS you are on you can just use 'jbang it.mulders:mcs:RELEASE' or 'jbang mcs@mthmulders' (if mthmulders/jbang-catalog repo gets created)

It does not require anything special- just publish the .jar at a stable location like the other binaries. 

Wdyt?

**Comment by mthmulders** [comment]
I think that's a great idea. Initially, I went for native binaries (for fastest launch) but providing binaries for other platforms is pretty limited by what GitHub Actions provide and/or what I have available.

**Comment by mthmulders** [comment]
@all-contributors please add @maxandersen for ideas.

**Comment by maxandersen** [comment]
Anything preventing publish of the jar in github releases or even maven central ?

**Reply by mthmulders** [reply]
From a technical perspective, not at all. The only thing preventing this is the amount of time I have on my hands, combined with not knowing where to start :-). I'd be grateful for any pointers in the right direction!
